integration-tests/integration_test.go

```diff
@@ -768,24 +778,38 @@ func (s *ProcessListeningOnPortTestSuite) TestProcessListeningOnPort() {
 	endpoints, err := s.GetEndpoints(s.serverContainer)
 	s.Require().NoError(err)
 
-	if !assert.Equal(s.T(), 4, len(endpoints)) {
+	if !assert.Equal(s.T(), 5, len(endpoints)) {
```

Comment:
The changes in this test are quite extensive and hide a lot of additional test cases amongst the existing test. I think it would be a bit cleaner to separate these out into their own test specifically for the symlink case

Reply:
Done.